scripts/ui/inventory/ui.gd

```diff
@@ -0,0 +1,46 @@
+extends CanvasLayer
+
+var holding_item = null
+
+# Called when the node enters the scene tree for the first time.
+func _input(event: InputEvent) -> void:
+	if event.is_action_pressed("Inventory"):
+		$Inventory.visible = !$Inventory.visible
+		$Hotbar.tooltip_change()
+	if event.is_action_pressed("scroll_up"):
+		if $Inventory.visible:
+			PlayerInventory.active_inventory_item_scroll_up()
+		else:
+			PlayerInventory.active_item_scroll_up()
+	elif event.is_action_pressed("scroll_down"):
+		if $Inventory.visible:
+			PlayerInventory.active_inventory_item_scroll_down()
+		else:
+			PlayerInventory.active_item_scroll_down()
+	if event.is_action_pressed("hotkey1"):
+		PlayerInventory.change_active_item_slot(0)
```

Comment:
There's a much simpler way to handle this kind of multi-slot key differentiation that's a little neater and faster, will discuss later.